### DISCUSSION TITLE
Irradiance map

### DISCUSSION THREAD

**Discussion by czpixel** [UMEP-dev/UMEP#253] [Q&A]
I need to be sure that I am getting the proper results please. Please find a screenshot attached. This is the energyyearroof.tif meaning a raster of irradiance on ground and building roofs named Energyyearroof.tif.  This should be a geoTIFF (Energyyearroof.tif) showing pixel wise total irradiance in kWh. Can I say that this is the irradiance map? I would need to filter out anything below 900kwh. Thanks for your help :)
![map72](https://user-images.githubusercontent.com/80916212/111675225-2ddfc180-881d-11eb-9673-6956347cc373.jpg)

**Comment by biglimp** [comment]
Your values look abit high. Please attach your meteorological file.

**Reply by czpixel** [reply]
Thanks for your reply!

The attached file (power_point_hourly) is the one I downloaded from https://power.larc.nasa.gov/beta/data-access-viewer/ 
And the rearranged.xlsx file is the one I used as an input into the UMEP tool (prepare existing data). I used the bare minimum possible. 
And the .txt is the output of prepare existing data.
Thanks for your help!


[Prepareweatherdata_rearranged.xlsx](https://github.com/UMEP-dev/UMEP/files/6169541/Prepareweatherdata_rearranged.xlsx)

[weatherfiletouseinput2.txt](https://github.com/UMEP-dev/UMEP/files/6169556/weatherfiletouseinput2.txt)

[POWER_Point_Hourly_20150101_20201231_035d9103N_014d3988E_LST - Copy.xlsx](https://github.com/UMEP-dev/UMEP/files/6169569/POWER_Point_Hourly_20150101_20201231_035d9103N_014d3988E_LST.-.Copy.xlsx)
 file)

**Reply by biglimp** [reply]
Your input data looks ok. Actually, where on the globe are you? We got similar values for a tropical location (Dar es Salaam) in a previous study (https://www.sciencedirect.com/science/article/pii/S2210670716304267).

Just as a test, just use one year of data and see if you get similar values.

**Reply by czpixel** [reply]
Thank you! I am on the island of Malta (Europe). I used one year of data and I got lower values (0-1200) unlike multiple years (up to 1989). So which one should I use? Why should values change if you use multiple years?

**Reply by biglimp** [reply]
There might be an issue in the code running multiple years. Can you please attach your **one year** and your **multiyear** weatherfile (as .txt) and  your **DSM**. Then I can try to see what is happening.

**Reply by czpixel** [reply]
I have attached the one year input file:
[one year 2018 weather.txt](https://github.com/UMEP-dev/UMEP/files/6188379/one.year.2018.weather.txt)
The multiple years:
[weatherfiletouseinput2.txt](https://github.com/UMEP-dev/UMEP/files/6188388/weatherfiletouseinput2.txt)
The DSM - this is a very small section, I am currently working on:
[results.48.zip](https://github.com/UMEP-dev/UMEP/files/6188406/results.48.zip)

Thanks for your help!

**Reply by czpixel** [reply]
I think I have displayed the wrong output file. Could it be ? I need to display the energyyearroof.tif. I am re-running all and I got low values again even with multiple years weather data. The site am currently working on isn't ideal. Did you get the same results? I am very new to qgis.

**Reply by biglimp** [reply]
My runs (1 year and 5 year) just finished. They show reasonable results, hence the model simulates multiple years correct. It is always better to run multiple years

**Reply by czpixel** [reply]
Would you mind attaching the results? (even a screenshot).
![mutliple years](https://user-images.githubusercontent.com/80916212/112158463-79f98000-8be8-11eb-8fb5-4b8450f1e952.jpg)
 I want to be sure that it is like mine please. Did you get same low values please?

**Reply by czpixel** [reply]
How would you interpret this (above screenshot) please? The maximum is 243 kwh per year?

**Reply by biglimp** [reply]
These are wrong. Have you ticked in ”estimate direct and diffuse radiation from global”? Have you set the UTC (+1) correct?

**Reply by biglimp** [reply]
This is what I got running with 5 years of data:

![image](https://user-images.githubusercontent.com/11908356/112277511-20de2a80-8c82-11eb-869c-f7ef901a5b4b.png)

**Comment by czpixel** [comment]
Thanks for your email. I have attached the requested files online (too
large via email)!
Do you think there is something wrong with the weather data?
Thanks for your help.
Regards,
Charmaine

**Comment by GLauretti** [comment]
Hello everybody, 
I would like to join the discussion because I am having the same problem.
I'm working on a solar cadaster about my University in Rome.
In Rome, the amount of solar radiation in 1 year is near 1600 kWh/m2 but the results from my SEBE analysis is than lower (1200, considering that the buildings are completely isolated all around from the rest of the city) when I count the energy on the roofs (without considering the slope), and much higher when I take the slope in account ('till 2200).
I took the data from my university meteorological station.
I thank you in advance for any advices.
Regards
Giulia Lauretti

**Reply by biglimp** [reply]
Could you attach you data (meteorological file and DSM) so that I can have a look.

**Reply by GLauretti** [reply]
This is one of the area of interest that I am studying.
Thank you for the fast support.
Dataset:
[dati_completi_2017.txt](https://github.com/UMEP-dev/UMEP/files/6181108/dati_completi_2017.txt)
DSM file:
[Rome_university_DSM.zip](https://github.com/UMEP-dev/UMEP/files/6181148/Rome_university_DSM.zip)

**Reply by biglimp** [reply]
Your time resolution is three hour but SEBE only works with hourly data. That is probably the why you see low irradiance values. The reason of using hourly resolution is distribute the incoming radiation more evenly on the sky vault.

**Comment by biglimp** [comment]
Ok, now I saw your answer in github. Lets continue discussion there.

Fredrik

23 mars 2021 kl. 17:13 skrev czpixel ***@***.***>:

﻿

How would you interpret this (above screenshot) please? The maximum is 243 kwh per year?

**Comment by czpixel** [comment]
Thanks for your reply.
When I run the tool, I get two .tif files.
Which one should I use? Is it the energyearroof.tif or dsm.tif ?

**Reply by biglimp** [reply]
energyearroof.tif

**Comment by czpixel** [comment]
Thank you so much!!! :) :) :) :) I didn't tick estimate direct and diffuse
radiation from global, so that is why I was getting the wrong values.
I have some final questions to ask you if that is OK.

   - I left albedo as 0.15
   - UTC 1
   - when analysing the height (to produce height raster) using urban
   geometry (UMEP tool), what is the wall limit that I should set please?
   - In addition, when preparing the weather file, the incoming shortwave
   radiation is also known as Top-Of-Atmosphere Shortwave Downward
   Irradiance (W/m^2) or All Sky Surface Shortwave Downward Irradiance (W/m^2)
   please or Clear Sky Surface Shortwave Downward Irradiance (W/m^2) ?
   - the values that I obtain from ernegyearroof.tif are kwh per year only
   or per year per m2 ?

Thank you very much, I really appreciate your help!

**Reply by biglimp** [reply]
- Albedo, ok
- I suspect that you have used a Lidar dataset to produce your dsm, hence your walls are somewhat unclear. Thus, maybe use a lower threshold like 2 meters. This only affects the wall irradiance and not energyyearroof.tif
- All Sky Downward Irradience, i.e., radiation at the Earth surface.
- Per year per m2

**Comment by czpixel** [comment]
Thank you!! Very helpful :)
